node/service/src/lib.rs

```diff
@@ -775,9 +776,22 @@ where
 
 					let author = nimbus_primitives::InherentDataProvider;
 
-					Ok((time, parachain_inherent, author))
+					let randomness = session_keys_primitives::InherentDataProvider;
+
+					Ok((time, parachain_inherent, author, randomness))
 				}
 			};
+			let client_clone = client.clone();
+			let keystore_clone = keystore.clone();
+			let additional_digests_provider = move |nimbus_id: NimbusId, parent: Hash|
```

Comment:
a less generic name would make this block of code more intuitive

Reply:
changed it to `maybe_provide_vrf_digest`